web/src/components/tasks/LicenseTask.tsx

```diff
@@ -133,6 +133,7 @@ export const LicenseTask = (props: Props): ReactElement => {
             <TabPanel value="0" sx={{ paddingX: 0 }}>
               <div className="margin-top-3">
                 <UnlockedBy task={props.task} />
+                <Content>{props.task.summaryDescriptionMd || ""}</Content>
```

Comment:
This line is necessary, otherwise the summary does not show up in the license tasks that are integrated with consumer affairs (cosmetology, HIC, etc)